drivers/interrupt_controller/Kconfig.esp32

```diff
@@ -5,7 +5,7 @@
 
 config INTC_ESP32
 	bool "Interrupt allocator for Xtensa-based Espressif SoCs"
-	default y if SOC_SERIES_ESP32
+	default y if SOC_FAMILY_ESPRESSIF_ESP32 && !SOC_SERIES_ESP32C3 && !SOC_SERIES_ESP32C6
 	help
```

Comment:
you changed this **to** `SOC_SERIES_ESP32` in an earlier commit in this PR and change it again, so it should be fixed in the original commit instead of changing it to something wrong then fixing it